modules/level_set/examples/rotating_circle/tests

```diff
@@ -0,0 +1,21 @@
+[Tests]
+  [./level_set_only]
+    type = RunApp
+    input = circle_rotate.i
+    cli_args = Executioner/num_steps=2
+    method = opt # Can be slow in debug and this is simply to make sure input file remains working
```

Comment:
I would rather run a few timesteps, there are multiapps in some of these and I want to make sure they run as well.